client/ayon_applications/plugins/publish/integrate_app_name.py

```diff
@@ -0,0 +1,10 @@
+import pyblish.api
+
+
+class IntegrateAsset(pyblish.api.InstancePlugin):
```

Comment:
Please specify a more unique or accurate plug-in class name :)

```suggestion
class IntegrateVersionAppNameData(pyblish.api.InstancePlugin):
```

Or whatever makes sense - just to avoid potential conflicts with duplicate class names?

Also, does `appName` still refer to `maya/2026` or alike when publishing on the farm.